checkov/terraform/checks/resource/aws/BatchJobIsNotPrivileged.py

```diff
@@ -14,6 +14,8 @@ def __init__(self):
     def scan_resource_conf(self, conf):
         if conf.get("container_properties"):
             if type(conf.get("container_properties")[0]) is str:
+                if "file(" in conf.get("container_properties")[0]:
+                    return CheckResult.UNKNOWN
                 container = json.loads(conf.get("container_properties")[0])
```

Comment:
```suggestion
                try:
                    container = json.loads(conf.get("container_properties")[0])
                except json.JSONDecodeError as e:
                    return CheckResult.UNKNOWN
```
then also other things will be ignored, if we couldn't process them before